mathesar_ui/src/systems/table-view/table-inspector/CollapsibleHeader.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { Dropdown, Icon } from '@mathesar/component-library';
+  import { iconDatabase } from '@mathesar/icons';
+
+  export let title: string;
+  export let isDBLevelConfiguration = false;
+</script>
+
+<div class="container">
+  <span>{title}</span>
+  {#if isDBLevelConfiguration}
+    <Dropdown
+      ariaLabel="Help"
+      placement="auto"
+      showArrow={false}
+      triggerAppearance="ghost"
+      triggerClass="help-trigger"
+      contentClass="help-content"
+    >
```

Comment:
Why is the help button a dropdown?

As per the design, it should be a tooltip, which appears on hover. @ghislaineguerin Am I correct in assuming this?

This help content currently only appears when clicking on the database icon.